src/worker/plugins/run.ts

```diff
@@ -27,6 +31,14 @@ export async function runPlugins(server: PluginsServer, event: PluginEvent): Pro
         }
     }
 
+    if (pluginsRan.length > 0 || pluginsFailed.length > 0) {
+        event.properties = {
+            ...event.properties,
+            $plugins_ran_successfully: pluginsRan,
```

Comment:
How about naming this `$plugins_succeeded`?

Reply:
I like it. Changed.